gammapy/spectrum/tests/test_spectrum_analysis.py

```diff
@@ -60,28 +62,28 @@ def test_spectral_fit(tmpdir):
     fit.energy_threshold_low = '100 GeV'
     fit.energy_threshold_high = '10 TeV'
     fit.run(method='sherpa')
-    assert_allclose(fit.model.gamma.val, 2.0, rtol=1e-1)
+    assert_allclose(fit.model.gamma.val, 2.23, rtol=1e-1)
```

Comment:
It's great to now have first high-level tests passing on travis-ci:
https://travis-ci.org/gammapy/gammapy/jobs/101052536#L1449

For spectral index, I'd find `atol` more intuitive than `rtol`. Can you please change to `atol` here?